src/app/identity.service.ts

```diff
@@ -83,6 +85,8 @@ export class IdentityService {
   }
 
   get(id: string, jwt: string, {withTipBalances = false}: {withTipBalances?: boolean} = {}): Observable<Person> {
+    this.cashBalanceStale = false;
+
```

Comment:
should we not wait until the person is returned from the server and then set the stale flag to false? I haven't tested but I think that would be:

```typescript
  get(id: string, jwt: string, {withTipBalances = false}: {withTipBalances?: boolean} = {}): Observable<Person> {
     return this.http.get<Person>(
      `${environment.identityApiPrefix}${this.peoplePath}/${id}`,
      {
        headers: new HttpHeaders({ 'X-Tbg-Auth': jwt }),
        params: {
          withTipBalances: withTipBalances ? 'true' : 'false',
        },
      },
    ).pipe(
      tap(() => {this.cashBalanceStale = false})
     );
  }
```